res/flow-capture.yml

```diff
@@ -20,7 +20,7 @@ spec:
       containers:
       - name: netobserv-cli
         # image: quay.io/netobserv/netobserv-ebpf-agent:main
-        image: quay.io/jpinsonn/netobserv-ebpf-agent:flow-stream-1
+        image: quay.io/jpinsonn/netobserv-ebpf-agent:1497-1
```

Comment:
This is a temporary image until https://issues.redhat.com/browse/NETOBSERV-1471 is done.

Reply:
Related code: https://github.com/netobserv/flowlogs-pipeline/pull/604